pandora-client-web/src/components/wardrobe/wardrobe.scss

```diff
@@ -265,8 +265,15 @@ $drop-overlay-gap: 0.3em;
 
 	.grid {
 		display: grid;
-		grid-template-columns: repeat(3, minmax(0, 384px));
 		grid-template-rows: min-content auto;
+
+		&.small {
+			grid-template-columns: repeat(3, minmax(0, 384px));
+		}
+
+		&.large {
+			grid-template-columns: repeat(4, minmax(0, 384px));
+		}
```

Comment:
why not just do a media quey instead?

Reply:
Because.... I did not even know that something like that exists. *giggles*
I looked it up just now and committed a change. I like it, as it now changes the layout dynamically on window changes and not only on react state changes. Thank you!~